client/scss/styleguide/_styleguide.scss

```diff
@@ -185,4 +66,53 @@
   }
 }
 
+.font {
```

Comment:
Again - I thought we were meant to be sharing as much as possible - having a `.styleguide` namespace means we're maintaining 2 systems, defeating the point of the exercise?

Unless we really only want them to share `HTML`?